app/views/govuk_publishing_components/components/_contextual_breadcrumbs.html.erb

```diff
@@ -1,47 +1,16 @@
-<% navigation = GovukPublishingComponents::Presenters::ContextualNavigation.new(content_item, request) %>
 <% prioritise_taxon_breadcrumbs ||= false %>
+<% breadcrumb_selector = GovukPublishingComponents::Presenters::BreadcrumbSelector.call(content_item, request, prioritise_taxon_breadcrumbs) %>
 <% inverse ||= false %>
 <% collapse_on_mobile ||= true unless local_assigns[:collapse_on_mobile].eql?(false) %>
 
 <div class='gem-c-contextual-breadcrumbs'>
-
-  <% if navigation.priority_taxon %>
-    <%= render 'govuk_publishing_components/components/step_by_step_nav_header',
-      {
-        title: navigation.priority_taxon['title'],
-        path: navigation.priority_taxon['base_path']
-      }
-    %>
-  <% elsif navigation.content_tagged_to_current_step_by_step? %>
-    <%# Rendering step by step nav breadcrumbs because there's 1 step by step %>
+  <% if breadcrumb_selector[:step_by_step] %>
     <%= render 'govuk_publishing_components/components/step_by_step_nav_header',
-      navigation.step_nav_helper.header %>
-  <% elsif navigation.content_tagged_to_a_finder? %>
-    <%# Rendering finder breadcrumbs because the page is tagged to a finder %>
-    <%= render 'govuk_publishing_components/components/breadcrumbs', breadcrumbs: navigation.breadcrumbs, inverse: inverse, collapse_on_mobile: collapse_on_mobile %>
-  <% elsif navigation.content_is_tagged_to_a_live_taxon? && prioritise_taxon_breadcrumbs %>
-    <%# Rendering taxonomy breadcrumbs because the page is tagged to live taxons
-          and we want to prioritise them over all other breadcrumbs %>
+               breadcrumb_selector[:breadcrumbs] %>
+  <% elsif breadcrumb_selector[:breadcrumbs] %>
     <%= render 'govuk_publishing_components/components/breadcrumbs',
-               breadcrumbs: navigation.taxon_breadcrumbs[:breadcrumbs],
+               breadcrumbs: breadcrumb_selector[:breadcrumbs],
                inverse: inverse,
```

Comment:
How does this get set for when we want it to be true?

Reply:
This is existing code. I believe it is set when the partial is called so to set it to inverse true it would be called via:
```ruby
<%= render 'path/to/contextual_breadcrumbs', .... , inverse: true %>
```